app/store/charmstore-api.js

```diff
@@ -345,6 +345,43 @@ YUI.add('charmstore-api', function(Y) {
           this.apiPath, charmId, 'icon.svg'].join('/');
       }
       return path;
+    },
+
+    /**
+      Takes the bundle id and fetches the bundle YAML contents. Required for
+      deploying a bundle via the deployer.
+
+      @method getBundleYAML
+      @param {String} id Bundle id in apiv4 format.
```

Comment:
I might make it clear we're talking charmstore here, e.g. `... id in v4 charmstore api format`.